triagebot.toml

```diff
@@ -174,8 +174,8 @@ label = "O-emscripten"
 
 [ping.relnotes-interest-group]
 message = """\
-Hi relnotes-interest-group, this PR adds release notes. Could you review this PR
-if you have time? Thanks <3
+Hi relnotes-interest-group, this relnotes issue/PR could use some help in
+reviewing / adjusting relnotes. Could you take a look if available? Thanks <3
```

Comment:
How about this tweak?
```suggestion
Hi relnotes-interest-group, this issue/PR could use some help in
reviewing / adjusting release notes. Could you take a look if available? Thanks <3
```

Reply:
Sure. (Incredible diff colors)

![Screenshot 2025-03-25 042452](https://github.com/user-attachments/assets/cd844bc1-e11c-46e2-8841-73e34e49c3da)

Reply:
Fixed